static/app/components/gridEditable/index.tsx

```diff
@@ -45,6 +45,7 @@ export type GridColumn<K = ObjectKey> = {
 
 export type GridColumnHeader<K = ObjectKey> = GridColumn<K> & {
   name: string;
+  tooltip?: string;
```

Comment:
This is bad, as I'm adding the definition here, but `gridEditable` doesn't use this prop. Ideally it would support tooltips out of the box, I can take a look into this in a separate PR.